internal/nodes/avi_model_l4_translator.go

```diff
@@ -470,10 +474,12 @@ func (o *AviObjectGraph) BuildL4LBGraph(namespace string, svcName string, key st
 		return
 	}
 	VsNode = o.ConstructAviL4VsNode(svcObj, key)
-	o.ConstructAviL4PolPoolNodes(svcObj, VsNode, key)
-	o.AddModelNode(VsNode)
-	utils.AviLog.Infof("key: %s, msg: checksum  for AVI VS object %v", key, VsNode.GetCheckSum())
-	utils.AviLog.Infof("key: %s, msg: computed Graph checksum for VS is: %v", key, o.GraphChecksum)
+	if VsNode != nil {
+		o.ConstructAviL4PolPoolNodes(svcObj, VsNode, key)
+		o.AddModelNode(VsNode)
+		utils.AviLog.Infof("key: %s, msg: checksum  for AVI VS object %v", key, VsNode.GetCheckSum())
+		utils.AviLog.Infof("key: %s, msg: computed Graph checksum for VS is: %v", key, o.GraphChecksum)
```

Comment:
Comment on the existing code. `o.GraphChecksum` would return 0. I think we should replace this with `vsNode.GraphChecksum`.

Reply:
I am resolving this comment. We can take this up later releases.